### PR TITLE
Adds dynamic reference indices to ISIS network writes

### PR DESCRIPTION
**Pull request by jlaura**
Reference indices can now be pulled from the data frame. PR https://github.com/USGS-Astrogeology/autocnet/pull/527 (commit is 'Fixes #524') adds the writes for dynamic reference indices into autocnet.

### CODE REVIEW

File: plio/io/io_controlnetwork.py [hunk 1]
```diff
@@ -335,6 +335,11 @@ def _set_pid(pointid):
             #point_spec.id = _set_pid(i)
             point_spec.id = _set_pid(i)
             point_spec.type = g.iloc[0].pointType
+            try:
+                point_spec.referenceIndex = g.iloc[0].referenceIndex
+            except:
+                warnings.warn('Unable to identify referenceIndex for points. Defaulting to index 0.')
```
Comment (jessemapel) [inline plio/io/io_controlnetwork.py:341]: It would be good to put the ID of the point in the warning string.

**Review by jessemapel — COMMENTED** [review]
Change looks good, but needs tests.

**Comment by jessemapel** [comment]
Fixes #111

**Comment by jlaura** [comment]
@jessemapel  This is ready for a re-review when you get a chance. I can peel a release once this looks good and conda-forge should then update I think.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**